tutorials/Kling_Gupta_efficiency.ipynb

```diff
@@ -0,0 +1,392 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0950f056",
+   "metadata": {},
+   "source": [
+    "# Kling–Gupta efficiency (KGE)\n",
+    "In this notebook, we will explore the Kling-Gupta Efficiency (KGE) which is widely used to evaluate the performance of hydrological models. KGE is a performance metric that decomposes the error into three components:\n",
+    "correlation,  variability and bias.\n",
+    "\n",
+    "It is computed as:\n",
+    "\n",
+    "$\n",
+    "\\text{KGE} = 1 - \\sqrt{\\left[s_\\rho \\cdot (\\rho - 1)\\right]^2 + \\left[s_\\alpha \\cdot (\\alpha - 1)\\right]^2 + \\left[s_\\beta \\cdot (\\beta - 1)\\right]^2}\n",
+    "$\n",
+    "\n",
+    "$\n",
+    "\\rho = \\frac{\\text{cov}(x, y)}{\\sigma_x \\sigma_y}\n",
+    "$\n",
+    "\n",
+    "$\n",
+    "\\alpha = \\frac{\\sigma_x}{\\sigma_y}\n",
+    "$\n",
+    "\n",
+    "$\n",
+    "\\beta = \\frac{\\mu_x}{\\mu_y}\n",
+    "$\n",
+    "\n",
+    "where\n",
+    "- $\\rho$ : Linear correlation between observed and forecast values.\n",
```

Comment:
```suggestion
    "where\n",
    "\n",
    "- $\\rho$ : Linear correlation between observed and forecast values.\n",
```
For dot points to render properly in both Jupyter Notebooks and readthedocs, you need an empty line above the dot points. 

![image](https://github.com/user-attachments/assets/4b68a3ad-eeaf-4628-830b-f7a5b8364484)

The image above is how it renders in readthedocs without the empty line.

![image](https://github.com/user-attachments/assets/637b2eda-4c28-406f-80b7-3db6c5913ff5)

The image above is how it renders in readthedocs with the empty line.


(Note: an exception to this is when dot points are on a line immediately following a header - in those instances dot points render properly in both Jupyter Notebook and readthedocs).